styles/globals.css

```diff
@@ -116,3 +116,158 @@ body {
   margin: 20px;
   border: 1px solid var(--border-default);
 }
+
+@media screen and (max-width: 1024px) {
+  .global__heading--large {
+    font-size: calc(128px / 1.5);
```

Comment:
Why we use cal() to calculate the size, if we use the result directly than calculating, will it better for faster render speed?

Reply:
Technically speaking, using the result directly would be faster. However, since this is a one-time modification and to make it easier for other team members to review, I have decided to use calc.

For other team members: Please consider that if you need to review my modification, you would have to manually calculate the result 50 times to verify its accuracy. This would be a difficult task, and I want to ensure that my modification is easily understandable and verifiable.

For myself: I hate math, so using calc will make it easier for me to complete the task accurately and efficiently. 

Additionally, using calc will make the code more readable for myself and other team members.